index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Cristian-Teddy Tribute Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+
+    <div id="main">
+      <div id="title">Dr. Norman Borlaug</div>
+
+      <div id="img-div">
+        <img id="image" src="image/norman-borlaug.jpg">
```

Comment:
- I suggest you add the `alt` attribute to every image rendered. This is because someone looking at the image can easily understand what's in it in case the image fails to load from the server. It also helps in SEO optimization.

Reply:
- [ ] You could also add as a reason, screens readers which is an assistive technology for the blind.